packages/modules/itwin-viewer-react/README.md

```diff
@@ -85,6 +85,25 @@ export const UserManagerHome = () => {
 };
 ```
 
+### Props
+
+#### Required
+
+- `projectId` - GUID for the project that contains the model that you wish to view
+- `iModelId` - GUID for the iModel that you wish to view
+- `authConfig` - an instance of an iModel.js FrontendAuthorizationClient or a function that returns an oidc-client UserManager
```

Comment:
Do you want to add a link the iModel.js docs for the `FrontendAuthorizationClient`?

Reply:
Good thought. I updated